post-processor.go

```diff
@@ -84,16 +87,36 @@ func (p *PostProcessor) Configure(raws ...interface{}) error {
 	}
 
 	var cred *credentials.Credentials = nil // nil credentials use the default aws sdk credential chain
-	// Setting either credential config variable indicates an attempt to use configured credentials
-	if p.config.AccessKey != "" || p.config.SecretKey != "" {
+
+	if p.config.AccessKey != "" && p.config.SecretKey != "" {
```

Comment:
That's what the code did previously, it checked that *both* the access key and the secret key were defined. With your change it's enough that any one of them is defined, the other can be empty.